### PR TITLE
empty 0.6.20b (new formula)

### PR DESCRIPTION
**Pull request by englishm**
Add empty - run processes under pseudo‐terminal sessions

<http://empty.sourceforge.net>

### CODE REVIEW

File: Library/Formula/empty.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Empty < Formula
+  desc "run processes under pseudo-terminal sessions"
+  homepage "http://empty.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/empty/empty/empty-0.6.20b/empty-0.6.20b.tgz"
+  sha256 "7e6636e400856984c4405ce7bd0843aaa3329fa3efd20c58df8400a9eaa35f09"
+
+  def install
+    inreplace "Makefile", "man/man1", "share/man/man1"
+    ENV.deparallelize
```
Comment (mikemcquaid) [inline Library/Formula/empty.rb]: Can you please submit this patch and the need to deparallelize to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

File: Library/Formula/empty.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    args = %W[
+      all
+      install
+      PREFIX=#{prefix}
+    ]
+    system "make", *args
```
Comment (mikemcquaid) [inline Library/Formula/empty.rb]: Just hard-code these args as they aren't dynamic.

File: Library/Formula/empty.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  end
+
+  test do
+    system "#{bin}/empty"
```
Comment (mikemcquaid) [inline Library/Formula/empty.rb]: Can you actually run a process here?

**Comment by bfontaine** [comment]
Ping?

**Comment by DomT4** [comment]
Closing as dead. Please do feel free to reopen if you wish to with the comments addressed. Thank you for your submission to Homebrew :heart:

**Pull request closed without merging**